app/static/js/searchUser.js

```diff
@@ -1,25 +1,39 @@
-export default function searchUser(inputId, callback, clear=false, parentElementId=null, columnRequested=null){
-  var query = $(`#${inputId}`).val()
+export default function searchUser(searchOptions){
```

Comment:
Don't replace the required arguments. Now there is no way to tell by looking at or calling this function what is required to make it work.